src/main/target/SITL/target.c

```diff
@@ -53,23 +57,38 @@ static pthread_t tcpWorker, udpWorker;
 static bool workerRunning = true;
 static udpLink_t stateLink, pwmLink;
 static pthread_mutex_t updateLock;
+static pthread_mutex_t mainLoopLock;
+
+int timeval_sub(struct timespec *result, struct timespec *x, struct timespec *y);
+
+int lockMainPID(void) {
+	return pthread_mutex_trylock(&mainLoopLock);
+}
 
 #define RAD2DEG (180.0 / M_PI)
-#define ACC_SCALE (256 / 9.81)
-#define GYRO_SCALE (8192 / 2000.0)
+#define ACC_SCALE (256 / 9.80665)
+#define GYRO_SCALE (16.4)
```

Comment:
because the value store as int16_t and then using `gyroDev_t.scale` scale back to float.